enterprise_gateway/itests/test_python_kernel.py

```diff
@@ -86,7 +86,7 @@ def test_get_deploy_mode(self):
 
     def test_get_host_address(self):
         result = self.kernel.execute("print(sc.getConf().get('spark.driver.host'))")
-        self.assertRegexpMatches(result, '\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}')
+        self.assertRegexpMatches(result, 'itest')
```

Comment:
We're going to want to be more flexible than this, since we should be able (and do) point at any cluster to run these tests against.  Since it appears spark changed the `spark.driver.host` property to be the actual hostname, I guess there are two options (perhaps others):
1. See if there's an applicable `spark.driver.ip` property and keep the result regex the same.
2. Modify the regex to accept any valid hostname, which is probably close to `[A-z,0-9,_,-,.]`.

Reply:
Ok - thanks.  How about this?  Let's define a test-based env, say, `ITEST_HOST_PREFIX` that represents the _expected_ hostname prefix.  This variable should default to the value of `itest`.  Then update the assertions to ensure this prefix (which should be treated as a regular expression string) appears as a prefix to the returned value.

This way we can run against entirely different clusters (outside the images used for integration tests) and, assuming each node of the cluster has the same prefix, should be able to pass the tests by conveying the appropriate value for `ITEST_HOST_PREFIX`.